styles.css

```diff
@@ -0,0 +1,57 @@
+#welcome-section {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    flex-direction: column;
+    padding: 50px;
+    height: 100vh;
+    overflow: auto;
+    background-color: burlywood;
+}
+
+body {
+    display: flex;
+    flex-direction: column;
+}
+.navbar{
+    display: flex;
+    align-items: center;
+    justify-content: right;
+    position: fixed;
+
+}
+.navbar > a{
+    padding: 10px;
+}
+
+#project {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+}
+#footer{
+    align-items: right;
+}
+
+#project a {
+    margin: 20px;
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;
+}
+@media only screen and (min-width: 768px) {
+    /* For desktop: */
+    .col-1 {width: 8.33%;}
+    .col-2 {width: 16.66%;}
+    .col-3 {width: 25%;}
+    .col-4 {width: 33.33%;}
+    .col-5 {width: 41.66%;}
+    .col-6 {width: 50%;}
+    .col-7 {width: 58.33%;}
+    .col-8 {width: 66.66%;}
+    .col-9 {width: 75%;}
+    .col-10 {width: 83.33%;}
+    .col-11 {width: 91.66%;}
+    .col-12 {width: 100%;}
```

Comment:
I would suggest removing these lines of code as no such classes exist in your .html file that will make your code clean.